experiments/static/experiments/js/experiments.js

```diff
@@ -13,15 +13,35 @@ experiments = function() {
         },
         goal: function(goal_name) {
             var xhr = new XMLHttpRequest();
+            this.csrfToken = experimentsCsrfToken && experimentsCsrfToken();
             xhr.open('POST', '/experiments/goal/' + goal_name + '/');
             xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+            if(this.csrfToken) {
+                xhr.setRequestHeader('X-CSRFToken', this.csrfToken);
+            }
             xhr.onload = function() {
                 if (xhr.status !== 200) {
                     throw 'POST to "/experiments/goal/" failed. Returned status of ' + xhr.status;
                 }
             };
             xhr.send();
-        }
+        },
+        getCookie: function(name) {
+            var cookieValue = null;
+            if (document.cookie && document.cookie != '') {
```

Comment:
I can't think of a situation where the second test makes any difference, `&& document.cookie != ''`. Is it safe to remove it?

Reply:
yeah I think it's safe to remove it, `document.cookie` will be an empty string when there are no cookies present so it'll already be false which means the second validation is redundant.